superset-frontend/src/components/Select/SupersetStyledSelect.tsx

```diff
@@ -221,8 +221,11 @@ function styled<
     // Handle onPaste event
     if (onPaste) {
       const Input = components.Input || defaultComponents.Input;
-      // @ts-ignore (needed for passing `onPaste`)
-      components.Input = props => <Input {...props} onPaste={onPaste} />;
+      components.Input = props => (
+        <div onPaste={onPaste}>
+          <Input {...props} />
```

Comment:
Not sure if there are tests around this component... have you verified by hand that this still works properly or looked for a test?

Reply:
I have verified it by hand, but we also have unit tests of `OnPasteSelect` component, which uses exactly this component.